doc/api/string_decoder.md

```diff
@@ -41,7 +41,7 @@ const decoder = new StringDecoder('utf8');
 
 decoder.write(Buffer.from([0xE2]));
 decoder.write(Buffer.from([0x82]));
-console.log(decoder.end(Buffer.from([0xAC])));
+console.log(decoder.end(Buffer.from([0xAC]))); // Prints: €
```

Comment:
Isn't this already implied by the text preceding this code block?

Reply:
@mscdex It's implied, but I thought it would be nice to show the result. won't it be needed?